tooling/nargo_cli/tests/execution_success/conditional_regression_421/src/main.nr

```diff
@@ -2,13 +2,9 @@ fn main(a: u32, mut c: [u32; 4]) {
     //Issue reported in #421
     if a == c[0] {
         assert(c[0] == 0);
-    } else {
-        if a == c[1] {
-            assert(c[1] == 0);
-        } else  {
-            if a == c[2] {
-                assert(c[2] == 0);
-            }
-        }
+    } else if a == c[1] {
+        assert(c[1] == 0);
+    } else if a == c[2] {
+        assert(c[2] == 0);
```

Comment:
This looks like a result of the  internal AST being the same for both items, but I think it's a neat consequence nevertheless.